### ISSUE TITLE
FINALLY <3

### ISSUE THREAD

**Issue by Skillz7z** [MichaelHillcox/Echelon3#1]
Yea bro that's what i want to see something fucking new xD 

Now you can get something really good 

Good luck i will be checking for updates always and reporting it when its unstable cuz i can't install it right now xD

**Comment by MichaelHillcox** [comment]
It's using phalcon meaning you need to install the phalcon extention to your php instance.

**Comment by Skillz7z** [comment]
It will later work without phalcon right ?

**Comment by Skillz7z** [comment]
Cuz many stuff like this xD 


Warning: Phalcon\Mvc\View\Engine\Volt\Compiler::compileFile(/var/www/subs/echelon/xtreme/cache/_var_www_subs_echelon_xtreme_app_core_views_index.volt.php): failed to open stream: No such file or directory in /var/www/subs/echelon/xtreme/public/index.php on line 59
Volt directory can't be written
#0 [internal function]: Phalcon\Mvc\View\Engine\Volt\Compiler->compileFile('/var/www/subs/e...', '/var/www/subs/e...', false)
#1 [internal function]: Phalcon\Mvc\View\Engine\Volt\Compiler->compile('/var/www/subs/e...')
#2 [internal function]: Phalcon\Mvc\View\Engine\Volt->render('/var/www/subs/e...', Array, true)
#3 [internal function]: Phalcon\Mvc\View->_engineRender(Array, 'index', true, true, NULL)
#4 [internal function]: Phalcon\Mvc\View->render('index', 'index')
#5 /var/www/subs/echelon/xtreme/public/index.php(59): Phalcon\Mvc\Application->handle()
#6 {main}

**Comment by MichaelHillcox** [comment]
No, Phalcon is the framework I'm using to build Echelon. You'll find that the current code won't work. All you've got is a admin homepage and a user dashboard. I'll make an offical release and give install instructions closer to the time of the first release.
